packages/api/src/external/carequality/models/hie-directory-view.ts

```diff
@@ -0,0 +1,13 @@
+import { Sequelize } from "sequelize";
+import { CQDirectoryEntryViewModel } from "./cq-directory-view";
+import { BaseModel } from "../../../models/_default";
+
+export class HIEDirectoryEntryViewModel extends CQDirectoryEntryViewModel {
```

Comment:
What'll happen when we have `CwDirectoryEntryModel`?

Reply:
I'll rewrite the entire View.

Reply:
this is basically just a passthrough till then